Makefile.main

```diff
@@ -169,6 +169,14 @@ test-coverage:
 	fi; \
 	bash <(curl -s https://codecov.io/bash) $${args}; \
 
+validate-commit: assets
+	git status --untracked-files=no --porcelain | grep -qe '..*'; \
+	if  [ $$? -eq 0 ] ; then \
+		git diff|cat; \
+		echo >&2 "generated bindata is out of sync"; \
```

Comment:
bindata? or assets? this rules are not generic, this PR shouldn't be here.

Reply:
changed error message. 
I'm okay to keep this inside dashboard. But we use the same rule in sdk&dashboard. So David suggested to add it in CI repo.
Is it useful here or not - totally up to you.